config.toml

```diff
@@ -17,6 +17,12 @@ contentDir = "/content/"
 [params.logos]
 navbar = "cncf-color.png"
 
+[[params.social]]
```

Comment:
On the whole, these are meant to be add-as-you-please, rather than a set that a project has to fill out – regardless, LGTM on #50

Reply:
I wanted to add it as a way of inviting folks to participate in the starter too :)